### PR TITLE
Web driver initializer

### PR DESCRIPTION
**Pull request by Gitch14**

### CODE REVIEW

File: src/main/java/executor/service/model/ProxyNetworkConfig.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
+package executor.service.model;
+
+import java.util.Objects;
+
+public class ProxyNetworkConfig {
+    String hostname;
```
Comment (artemkedyk) [inline src/main/java/executor/service/model/ProxyNetworkConfig.java:6]: Add private modifier

File: src/test/java/executor/service/model/ProxyConfigHolderTest.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ProxyConfigHolderTest {
+
+    @BeforeEach
+    void setUp() {
```
Comment (artemkedyk) [inline src/test/java/executor/service/model/ProxyConfigHolderTest.java:11]: remove this method

File: src/test/java/executor/service/model/ProxyNetworkConfigTest.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
+package executor.service.model;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ProxyNetworkConfigTest {
```
Comment (artemkedyk) [inline src/test/java/executor/service/model/ProxyNetworkConfigTest.java:8]: remove this class

File: src/test/test.iml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (artemkedyk) [inline src/test/test.iml:1]: add this file to git ignore

File: src/main/java/executor/service/model/ProxyNetworkConfig.java (hunk 1 above)
Comment (Gitch14) [reply src/main/java/executor/service/model/ProxyNetworkConfig.java:6]: I fixed

File: src/main/java/executor/service/StepExecution/Property.java [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+                String password = property.getProperty("ProxyConfigHolder.ProxyCredentials.password");
+                String hostname = property.getProperty("ProxyConfigHolder.ProxyNetworkConfig.hostname");
+                String port = property.getProperty("ProxyConfigHolder.ProxyNetworkConfig.port");
+                String webDriverExecutable = property.getProperty("WebDriverConfig.webDriverExecutable");
+                String userAgent = property.getProperty("WebDriverConfig.userAgent");
+                String pageLoadTimeout = property.getProperty("WebDriverConfig.pageLoadTimeout");
+                String implicitlyWait = property.getProperty("WebDriverConfig.implicitlyWait");
+
+
+                System.out.println("username: " + username
```
Comment (artemkedyk) [inline src/main/java/executor/service/StepExecution/Property.java:30]: I recommend using logger instead of sout

**Pull request closed without merging**